src/components/Header.jsx

```diff
@@ -41,8 +41,6 @@ export default function Header({ address, web3Modal, loadWeb3Modal, logoutOfWeb3
     }
   }
 
-  const { currentTheme } = useThemeSwitcher();
```

Comment:
I think we are safe to delete this whole file. Afaik it is cruft from scaffold-eth

Reply:
You're right. Done!